code/src/index.css

```diff
@@ -1,13 +1,216 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Roboto Mono', monospace;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: rgb(255,173,173);
+  background: radial-gradient(circle, rgba(255,173,173,1) 35%, rgba(255,255,255,1) 100%);
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+h1 {
+font-size: 16px;
+line-height: 0px;
+font-weight: 200;
+display: flex;
+justify-content: center;
 }
+
+h4{
+  font-size: 18px;
+  font-family: 'Roboto Mono', monospace;
+  font-weight: 200;
+}
+
+p {
+  font-size:16px;
+  font-family: 'Roboto Mono', monospace;
+  margin: 10px 0px 5px 5px;
+}
+
+/*Styling for content in the first box: Thought input-box*/
+.thought-input-wrapper {
+  border: 1px solid #000;
+  box-shadow: 7px 7px #000;
+  width: 40vw;
+  height: auto;
+  margin: 30px auto;
+  margin-top: 8%;
+  padding: 0px 25px 5px 25px;
+  background-color: #F2F0F0;
+  position:relative;
+}
+
+.thought-form-details{
+ display:flex;
+ justify-content: space-between;
+}
+
+.character-counter{
+ padding-right: 5%;
+ color: rgb(178, 174, 174);
+}
+
+.send-thought-button {
+  border: none;
+  border-radius: 25px;
+  font-family: 'Roboto Mono', monospace;
+  font-size: 16px;
+  background: #ffadad;
+  padding: 10px 15px;
+  margin: 2px 0 15px 0;
+  outline-color: #ffadad;
+}
+
+.text-input-area {
+  width: calc(40vw - 50px);
+  height: 60px;
+  font-family: 'Roboto Mono',monospace;
+  padding: 10px;
+  margin: 0px auto 5px;
+  resize: none;
+  outline-color:#ffadad;
+}
+
```

Comment:
I love this thing with the outline-color! 😄 As I said before, small details make a huge difference.